UnitTests/BraintreeVenmoTests/BTVenmoDriver_Tests.swift

```diff
@@ -613,7 +613,6 @@ class BTVenmoDriver_Tests: XCTestCase {
         let venmoDriver = BTVenmoDriver(apiClient: apiClient)
         
         XCTAssertEqual(venmoDriver.apiClient.metadata.integration, BTClientMetadataIntegrationType.custom)
-        XCTAssertEqual(venmoDriver.apiClient.metadata.source, BTClientMetadataSourceType.venmoApp)
```

Comment:
Similarly, rename or remove?  Is there no default source now, and if so, will this bork analytics going forward?

Reply:
https://github.com/braintree/braintree_ios/pull/789#discussion_r814809470